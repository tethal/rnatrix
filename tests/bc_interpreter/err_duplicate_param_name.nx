fun f(a, a) {
}

fun main() {
    f(42, 43);
}