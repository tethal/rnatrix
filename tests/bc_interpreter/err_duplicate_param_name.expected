<string>:1:10: error: symbol a already defined in this scope
fun f(a, a) {
         ^